package com.practise.Exceptions;

public class A {

	int a =10;
	int b =0;
	int c;
	public void divide() throws ArithmeticException{
		throw new ArithmeticException("User Defined");
	}
	
	
}
